packages/protocol/lib/compatibility/verify-bytecode.ts

```diff
@@ -15,9 +15,6 @@ import Web3 from 'web3'
 import { ignoredContractsV9 } from './ignored-contracts-v9'
 
 let ignoredContracts = [
-  // This contract is not proxied
-  'TransferWhitelist',
```

Comment:
We should consider leaving this here since it is possible that someone will try to verify previous releases and since this contract is part of previous releases it might happen that verification will fail (or we should double check that it will not fail)

Reply:
Keeping the ignore seems the safe thing to do. I've added it back.